src/plugins/intel_gpu/src/plugin/compiled_model.cpp

```diff
@@ -178,22 +178,13 @@ std::shared_ptr<ov::IAsyncInferRequest> CompiledModel::create_infer_request() co
 //     [ ov::Node::Input/ ov::Node::Output ]
 //     [ ov::intel_gpu::Graph ]
 void CompiledModel::export_model(std::ostream& model) const {
-    // If ov::CacheMode::OPTIMIZE_SIZE is set, do the export iff it's possible to do weightless caching
-    // which requires the weights_path or ov::Model ptr.
-    ov::CacheMode cache_mode = m_config.get_cache_mode();
-    std::string weights_path = m_config.get_weights_path();
-    auto model_ptr = m_config.get_model();
-    if (cache_mode == ov::CacheMode::OPTIMIZE_SIZE && !ov::util::validate_weights_path(weights_path) &&
-        model_ptr == nullptr) {
-        return;
-    }
```

Comment:
Is it possible to create a weightless cache without `weights_path` and `model_ptr`?

Reply:
Yes, we can always create weightless cache. These properties (weights_path or model_ptr) are only needed during import and there are relevant checks in the import code.